### PR TITLE
[CNTRL-50] adding schemas and types

### PR DESCRIPTION
**Pull request by ArsenBatyuchok**

### CODE REVIEW

File: src/article/Article.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  type: ArticleItemType.Video;
+  commonParams: {
+    url: string;
+    sizing: string;
+    ratioLock: boolean;
+  };
+  layoutParams: Record<
+    Layout,
+    {
+      fullwidth: boolean;
```
Comment (roman-hado) [inline src/article/Article.ts:81]: isn't it camelCase - fullWidth?

File: src/article/ItemBase.schema.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod';
+import { ItemArea } from './ItemArea';
+
+export const Link = z.object({
+  url: z.string().url(),
+  target: z.string().min(1)
+});
+
+const CommonParamsBase = z.object({
+  sizing: z.string().min(1),
```
Comment (roman-hado) [inline src/article/ItemBase.schema.ts:10]: remove ,

File: src/article/RichTextItem.schema.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  })
+));
+
+export const RichTextItemSchema = ItemBaseSchema.extend({
+  type: z.literal(ArticleItemType.RichText),
+  commonParams: z.object({
+    sizing: z.string(),
+    text: z.string(),
+    blocks: z.array(RichTextBlock).optional(),
+    styles: z.array(RichTextStyle).optional(),
```
Comment (roman-hado) [inline src/article/RichTextItem.schema.ts:38]: remove ,

File: src/metaInfo/MetaInfo.schema.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { z } from 'zod';
+
+export const PageMetaSchema = z.object({
+  title: z.string().optional(),
+  description: z.string().optional(),
+  opengraphThumbnail: z.string().optional(),
+  keywords: z.string().optional(),
```
Comment (roman-hado) [inline src/metaInfo/MetaInfo.schema.ts:7]: remove ,

**Review by roman-hado — APPROVED** [review]

File: src/article/Article.ts (hunk 1 above)
Comment (ArsenBatyuchok) [reply src/article/Article.ts:81]: no, it's not

**Review by sergokiko — APPROVED** [review]

**Pull request merged**